src/doc/unstable-book/src/language-features/asm-experimental-arch.md

```diff
@@ -145,14 +151,19 @@ This feature tracks `asm!` and `global_asm!` support for the following architect
-| All          | `fr` (Hexagon), `$fp` (MIPS), `Y` (AVR), `r4` (MSP430), `a6` (M68k), `r11` (s390x), `x29` (Arm64EC) | The frame pointer cannot be used as an input or output.                                                                                                                             |
+| All          | `fr` (Hexagon), `fp` (PowerPC), `$fp` (MIPS), `Y` (AVR), `r4` (MSP430), `a6` (M68k), `r11` (s390x), `x29` (Arm64EC) | The frame pointer cannot be used as an input or output.                                                                                                                             |
 | All          | `r19` (Hexagon), `x19` (Arm64EC)        | This is used internally by LLVM as a "base pointer" for functions with complex stack frames.                                                                                        |
 | MIPS         | `$0` or `$zero`                         | This is a constant zero register which can't be modified.                                                                                                                           |
 | MIPS         | `$1` or `$at`                           | Reserved for assembler.                                                                                                                                                             |
 | MIPS         | `$26`/`$k0`, `$27`/`$k1`                | OS-reserved registers.                                                                                                                                                              |
 | MIPS         | `$28`/`$gp`                             | Global pointer cannot be used as inputs or outputs.                                                                                                                                 |
 | MIPS         | `$ra`                                   | Return address cannot be used as inputs or outputs.                                                                                                                                 |
 | Hexagon      | `lr`                                    | This is the link register which cannot be used as an input or output.                                                                                                               |
+| PowerPC      | `$r2`, `$r13`                           | These are system reserved registers.                                                                                                                                                |
+| PowerPC      | `$r29`, `$r30`                          | These are used internally by LLVM.                                                                                                                                                  |
```

Comment:
It's ok to not support these two registers conservasively IMO, might rephrase to `These might be used as base pointer inside LLVM`.

Reply:
I found a line that collectively describes registers in the same situation in other architectures, so I moved these two there.

https://github.com/rust-lang/rust/blob/ca59dfec2902a08f5921f5f16f6aa5c741466d0e/src/doc/unstable-book/src/language-features/asm-experimental-arch.md?plain=1#L155